indico/modules/events/contributions/models/contributions.py

```diff
@@ -248,3 +248,11 @@ def can_manage(self, user, role=None, allow_admin=True, check_parent=True, expli
                 session_settings.get(self.event_new, 'coordinators_manage_contributions')):
             return True
         return False
+
+    @classmethod
+    def get_location_for_parent(cls, event):
```

Comment:
`event` should be `parent` since it might be another LocationMixin-based object (such as a session slot)